src/inverseRobot.js

```diff
@@ -8,6 +8,17 @@
 
 function inverseRobot(robot) {
   // write code here
+  const rightRobot = {};
+
+  for (const key in robot) {
+    if (rightRobot.hasOwnProperty(robot[key])) {
```

Comment:
Use 'Object.prototype.hasOwnProperty.call' instead of calling 'hasOwnProperty' directly on the object to avoid potential issues with overwritten properties.